code/modules/cm_marines/dropship_ammo.dm

```diff
@@ -386,9 +386,6 @@
 	impact.ceiling_debris_check(2)
 	spawn(5)
 		cell_explosion(impact, 200, 44, EXPLOSION_FALLOFF_SHAPE_LINEAR, null, create_cause_data(initial(name), source_mob))
-		var/datum/effect_system/expl_particles/P = new/datum/effect_system/expl_particles()
-		P.set_up(4, 0, impact)
-		P.start()
```

Comment:
Why is this necessary? It just looks way less interesting now. Did you intend this to be just a part of the cell_explosion proc and forgot?

![old](https://github.com/user-attachments/assets/1805865e-697b-407d-bd06-3f0dcf2cc0b9)
![new](https://github.com/user-attachments/assets/c89ed995-82bc-41fd-b1a3-a0b796f8d40e)